dev.dockerfile

```diff
@@ -2,7 +2,7 @@
 # This file is intended to be used with ./docker-compose.yml #
 ##############################################################
 
-FROM node:10.14.1-alpine as build
+FROM node:12.16.3-alpine as build
```

Comment:
Is it our policy to always specify an exact version when pulling the docker image? This could auto-update to the most recent LTS version if you used the alias `node:lts-alpine`.

Reply:
We use exact working version to avoid any bugs in latest version might have.  In this way our CI env would be more stable.

Reply:
But I can have test pr to try `node:lts-alpine`  and track it for 10-15 days and then discuss the outcomes.